src/qt/coincontroldialog.cpp

```diff
@@ -686,19 +692,19 @@ void CoinControlDialog::updateLabels(WalletModel* model, QDialog* dialog)
     if (model && model->getOptionsModel())
         nDisplayUnit = model->getOptionsModel()->getDisplayUnit();
 
-    QLabel* l1 = dialog->findChild<QLabel*>("labelCoinControlQuantity");
-    QLabel* l2 = dialog->findChild<QLabel*>("labelCoinControlAmount");
-    QLabel* l3 = dialog->findChild<QLabel*>("labelCoinControlFee");
-    QLabel* l4 = dialog->findChild<QLabel*>("labelCoinControlAfterFee");
-    QLabel* l5 = dialog->findChild<QLabel*>("labelCoinControlBytes");
-    QLabel* l7 = dialog->findChild<QLabel*>("labelCoinControlLowOutput");
-    QLabel* l8 = dialog->findChild<QLabel*>("labelCoinControlChange");
+    QLabel* l1 = findChild<QLabel*>("labelCoinControlQuantity");
+    QLabel* l2 = findChild<QLabel*>("labelCoinControlAmount");
```

Comment:
Now that this method is inside the dialog object, can move all of these labels to the direct member access form `ui-> labelCoinControlQuantity` and not try to find the child with the hardcoded id + cast it.

Reply:
totally makes sense.
We can squash it in the first commit (which makes updateLabels a class function)